webapp/channels/src/components/edit_post/edit_post.tsx

```diff
@@ -512,6 +512,15 @@ const EditPost = ({editingPost, actions, canEditPost, config, channelId, draft,
             <div className='post-body__actions'>
                 {emojiPicker}
             </div>
+            {
+                editText.includes('@') && (
```

Comment:
This will trigger even when we edit a pre-existing message that already contains at-mentions. The approach I have in mind involves comparing the newly edited message to the original one and checking if there are any changes that involve at-mentions in the difference

Reply:
Hi @M-ZubairAhmed I have improved the logic.
Can you review the code again
Thanks